src/citrine/resources/ingredient_spec.py

```diff
@@ -14,7 +14,39 @@
 
 
 class IngredientSpec(DataConcepts, Resource['IngredientSpec'], TaurusIngredientSpec):
-    """An ingredient spec."""
+    """
+    An ingredient specification.
+
+    Ingredients annotate a material with information about its usage in a process.
+
+    Parameters
+    ----------
+    uids: Map[str, str], optional
+        A collection of unique identifiers, each a key-value pair. The key is the "scope"
+        and the value is the identifier. The scope "id" is reserved for the internal Citrine ID,
+        which will always be a uuid4.
+    tags: List[str], optional
+        A set of tags. Tags can be used for filtering.
+    notes: str, optional
+        Long-form notes about the ingredient spec.
+    material: MaterialSpec
+        Material that this ingredient is.
+    mass_fraction: ContinuousValue, optional
+        The mass fraction of the ingredient in the process
+    volume_fraction: ContinuousValue, optional
+        The volume fraction of the ingredient in the process
+    number_fraction: ContinuousValue, optional
+        The number fraction of the ingredient in the process
+    absolute_quantity: ContinuousValue, optional
+        The absolute quantity of the ingredient in the process
+    unique_label: str, optional
```

Comment:
This was brought up in the display_name -> name discussion.  Did we decide it should be migrated?

Reply:
See above